kubernetes-kit-starter/src/main/java/com/vaadin/kubernetes/starter/sessiontracker/SessionSerializer.java

```diff
@@ -231,6 +232,40 @@ public boolean isRunning() {
         return !stopped.get();
     }
 
+    /**
+     * Creates a new HTTP session associated with the given cluster key. This
+     * method attempts to mark the deserialization process as started and, if
+     * successful, creates a new session using the provided request.
+     *
+     * @param clusterKey
+     *            the distributed storage key associated with the session to be
+     *            created.
+     * @param request
+     *            the HttpServletRequest used to create the HTTP session.
+     * @return the newly created HTTP session, or {@code null} if the
+     *         deserialization process could not be started or an exception
+     *         occurred.
+     */
+    public HttpSession createHttpSession(String clusterKey,
+            HttpServletRequest request) {
```

Comment:
Do we need this new method to be public?

Reply:
Good point. I think we ca make it package protected